action.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Publish Regulatory artifacts"
+description: "Publishes the regulatory artifacts to the correct location for the SLT"
+
+runs:
+  using: composite
+  steps:
+    - name: Publish the test result file
+      shell: bash
+      run: |
+        # Check environment variables
+        if [ -z "$TEST_RESULTS_PATH" ]; then
+          TEST_RESULTS_PATH=test-reports/junit.xml
+        fi
+
+        if [ -z "$REGULATORY_S3_BUCKET" ]; then
+          echo "Missing the REGULATORY_S3_BUCKET environment variable!"
+          exit 1
+        fi
+
+        # Check if the test results file exist, skip if not
+        if [ ! -e "$TEST_RESULTS_PATH" ]; then
+          echo "Was not able to find $TEST_RESULTS_PATH"
+          echo "Skipping test result upload"
+          exit 0
+        fi
+
+        # Grab variables from the github context (https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)
+        repository="${{github.repository}}"             # For example "Codertocat/Hello-World"
```

Comment:
(Question) Is it needed to store the Github variable in a bash variable?

Reply:
Yes, good question. For the "sub string construction" (`${repository:$repository_characters_to_skip}`) to work a few lines down, there needs to be a bash variable. Aside from that, I think it is also a bit easier to read and adjust later down the line.